packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -418,6 +418,20 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange>
 		const enrichedCommit = this.resubmitMachine.peekNextCommit();
 		this.submitCommit(enrichedCommit, localOpMetadata, true);
 	}
+	public rollback(content: JsonCompatibleReadOnly, localOpMetadata: unknown): void {
```

Comment:
Do calls to rollback always come in reverse order? That is, later ops are always rolled back before earlier ops? I assume that's the case, or I'd expect the below code to probably error, but just double checking.

Reply:
yeah. that is always the case. most dds require it